src/main/java/com/mongodb/jdbc/MongoDriver.java

```diff
@@ -59,6 +59,7 @@ public class MongoDriver implements Driver {
     static final String MONGODB_SRV_URL_PREFIX = JDBC + "mongodb+srv:";
     public static final String USER = "user";
     public static final String PASSWORD = "password";
+    public static final String CLIENT_INFO = "ClientInfo";
```

Comment:
```suggestion
    public static final String CLIENT_INFO = "clientInfo";
```
[nit] the other options start with lower case letters -- should we follow that pattern here as well?

Reply:
Sounds good, I'll make the change.